### PR TITLE
Merge express into master

### PR DESCRIPTION
**Pull request by bradmanAWS**
**Issue #:** 

**Description of changes:**
Hold until MCM for release

**Testing**
1. Have you successfully run `npm run build:release` locally?
Yes
2. How did you test these changes?
local build
3. If you made changes to the component library, have you provided corresponding documentation changes?
n/a
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -23,19 +27,47 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added Echo icon
 - Added poorConnection property to DeskPhone icon
 - Added optional 'id' prop for ui components
+- Added optional `timestamp` prop in ChatBubble
+- Added forwardRef for ChatBubbleContainer
+- Added optional img to MessageAttachment
```
Comment (nicklera1) [inline CHANGELOG.md:32]: all of this duplicated couple of lines below

File: CHANGELOG.md [hunk 2]
```diff
@@ -23,19 +27,47 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
+### Removed
+
+- Removed content in ChatBubble
+- Removed showName in ChatBubble
 - Change control bar theme opacity to 1
+- Changed RosterHeader 'title' prop to all for elements as well as strings
+- Changed senderName to optional in ChatBubble
+- Moved children inside of a div in ChatBubble
+- Changed `MeetingManager` to strictly enforce `DevicePermissionStatus` type.
```
Comment (nicklera1) [inline CHANGELOG.md:64]: Duplicates from 10lines above

**Comment by p-foucht** [comment]
LGTM aside from Nick's comments

File: CHANGELOG.md (hunk 2 above)
Comment (bradmanAWS) [reply CHANGELOG.md]: Good catch. I had to comb through these changelog conflicts like 10 times.

**Review by p-foucht — APPROVED** [review]

**Pull request merged**